test/cli_test.go

```diff
@@ -103,17 +109,29 @@ func TestCLI(t *testing.T) {
-		"TestCLI/v=0.12/tf=example_storage_bucket_iam_member_random_suffix/offline=false":                         "test produces inconsistent results based on randomized names - github.com/GoogleCloudPlatform/terraform-validator/issues/259",
-		"TestCLI/v=0.12/tf=example_storage_bucket_iam_member_random_suffix/offline=false/cmd=convert":             "test produces inconsistent results based on randomized names - github.com/GoogleCloudPlatform/terraform-validator/issues/259",
+		"TestCLI/v=0.12/tf=example_compute_forwarding_rule/offline=true/cmd=convert":                                   "temperarily skip because of the predictable drift in offline mode",
+		"TestCLI/v=0.12/tf=example_compute_forwarding_rule/offline=true/cmd=validate/constraint=always_violate":        "temperarily skip because of the predictable drift in offline mode",
+		"TestCLI/v=0.12/tf=example_compute_instance/offline=true/cmd=convert":                                          "compute_instance doesn't work in offline mode - github.com/hashicorp/terraform-provider-google/issues/8489",
+		"TestCLI/v=0.12/tf=example_compute_instance/offline=true/cmd=validate/constraint=always_violate":               "compute_instance doesn't work in offline mode - github.com/hashicorp/terraform-provider-google/issues/8489",
+		"TestCLI/v=0.12/tf=example_organization_iam_binding/offline=false/cmd=convert":                                 "skip because test runner doesn't have org permissions",
+		"TestCLI/v=0.12/tf=example_organization_iam_binding/offline=false/cmd=validate/constraint=always_violate":      "skip because test runner doesn't have org permissions",
+		"TestCLI/v=0.12/tf=example_organization_iam_member/offline=false/cmd=convert":                                  "skip because test runner doesn't have org permissions",
+		"TestCLI/v=0.12/tf=example_organization_iam_member/offline=false/cmd=validate/constraint=always_violate":       "skip because test runner doesn't have org permissions",
+		"TestCLI/v=0.12/tf=example_project_iam/offline=false/cmd=convert":                                              "example_project_iam is too complex to untangle merges with online data generically",
+		"TestCLI/v=0.12/tf=example_storage_bucket_iam_member_random_suffix/offline=false":                              "test produces inconsistent results based on randomized names - github.com/GoogleCloudPlatform/terraform-validator/issues/259",
+		"TestCLI/v=0.12/tf=example_storage_bucket_iam_member_random_suffix/offline=false/cmd=convert":                  "test produces inconsistent results based on randomized names - github.com/GoogleCloudPlatform/terraform-validator/issues/259",
+		"TestCLI/v=0.12/tf=example_kms_key_ring_iam_binding/offline=false/cmd=convert":                                 "skip because test keyring does not exist in project",
```

Comment:
These tests shouldn't be skipped - the terraform file needs to be written a little differently instead. I'll explain in more detail in a comment on one of the terraform files.

Reply:
@melinath , this is fixed